public/css/email_l10n_testing.css

```diff
@@ -0,0 +1,176 @@
+body.email-locale-testing {
+  background: #ededf0;
+  background-color: #ededf0;
+  background-image: none;
+  color: rgba(12, 12, 13, 0.7);
+}
+
+body.email-locale-testing .section-wrapper {
+  justify-content: center;
+  position: relative;
+  flex-wrap: nowrap;
+}
+
+.email-test-wrapper {
+  width: 600px;
+}
+
+.email-locale-testing nav {
+  margin: 30px;
+}
+
+.email-locale-testing nav a {
+  color: #000;
+  display: block;
+  font-size: 13px;
+  margin: 10px 0;
+}
+
+.email-logo,
+.email-body,
+.email-footer {
+  padding: 30px;
+}
+
+.email-content {
+  overflow: hidden;
+  border-radius: 3px;
+  box-shadow: 2px 2px 5px #7788995c;
```

Comment:
Not sure on your feelings on #RRGGBBAA versus `rgba(rr, gg, bb, aa)` above on L5. Not sure if one has better cross-browser support, or something.

Reply:
Oops- RGBA is the clear winner according to canIuse. Thanks!